CMakeLists.txt

```diff
@@ -35,6 +35,11 @@ else()
     endif()
 endif()
 
+
+if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang" OR "${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -pedantic") # if compiler is GCC or Clang add warning flags
+endif()
+
```

Comment:
Sure I understand. Thanks for your answer. I will remove theses changes and solve the warning issues in my project since I have these compilation flags. I do not think they are necessary either.